common/src/test/java/co/elastic/otel/common/ChainingSpanProcessorAutoConfigurationTest.java

```diff
@@ -106,9 +107,11 @@ protected boolean requiresStart() {
 
       List<SpanProcessor> spanProcessors = OtelReflectionUtils.getSpanProcessors(otel);
       assertThat(spanProcessors)
-          .containsExactlyInAnyOrder(
-              chainingProcessor.get(), SpanProcessor.composite() // NOOP-processor
-              );
+          .hasSize(3)
```

Comment:
You can revert these changes I think after removing the span-stacktrace processor from the test classpath.